lib/generatePrint.js

```diff
@@ -0,0 +1,47 @@
+const kebab = require('kebab-case')
+const classPostfix = require('postcss-class-postfix')
+const postcss = require('postcss')
+
+const generators = require('./generators')
+
+const RELEVANT_PRINT_GENERATORS = [
+  'spacing',
+  'clears',
+  'display',
+  'flexbox',
+  'floats',
+  'fontStyle',
+  'sizes',
+  'letterSpacing',
+  'lineHeight',
+  'maxWidths',
+  'textDecoration',
+  'textAlign',
+  'textTransform',
+  'typeScale',
+  'typography',
+  'fontWeight',
+  'visibility',
+  'whiteSpace',
+  'widths',
+  // add the @page specific generator
+  'page'
+]
+
+const requirePartial = name => {
+  const kebabed = kebab(name)
+  return require(`../partials/_${kebabed}.css`)
+}
+
+module.exports = async (config, key) => {
+  const css = RELEVANT_PRINT_GENERATORS.map(name => {
+    if (name in generators) {
+      return generators[name].css(config[name], config)
+    }
+    return requirePartial(name)
+  }).join('\n')
+
+  return postcss([classPostfix(`-${key}`)])
```

Comment:
shouldn't it be "suffix"?

Reply:
I agree, but... it's the name of the package `postcss-class-postfix` 😆